modules/api-svc/src/config.js

```diff
@@ -92,27 +92,27 @@ module.exports = {
     backendEventHandler: {
         enabled: env.get('ENABLE_BACKEND_EVENT_HANDLER').default('true').asBool(),
         domainEventConsumer: {
-            brokerList: env.get('BACKEND_EVENT_CONSUMER_BROKER_LIST').default('localhost:9092').asString(),
+            brokerList: env.get('BACKEND_EVENT_CONSUMER_BROKER_LIST').default('kafka:9092').asString(),
```

Comment:
I would prefer these to default to `local` and rather have them set to `kafka` as part of integration tests, etc.

Same comment applies elsewhere.